core/templates/pages/exploration-editor-page/editor-navigation/editor-navigation.component.html

```diff
@@ -1,5 +1,5 @@
+<i class="fa fa-book show-mobile-navbar-icon e2e-test-mobile-options" (click)="toggleMobileNavOptions()" [ngClass]="{'icon-move': mobileNavOptionsAreShown}" aria-label="Mobile navbar options" role="button"></i>
```

Comment:
`icon-move` is not a clear name, please rename the class to something such as `mobile-navbar-toggled`

Reply:
Renamed.